app/javascript/controllers/concepts_json.js

```diff
@@ -0,0 +1,41 @@
+import {Controller} from "@hotwired/stimulus"
+
+export default class extends Controller {
+    static values = {
+        ontologyId: String,
+        conceptId: String,
+        apiKey: String
+      }
+
+    update(){
+        const baseClassUrl = `${this.ontologyIdValue}/classes/${this.conceptIdValue}`;
+        const apikey = `apikey=${this.apiKeyValue}`;
+        const tabsList = document.querySelectorAll('#concept_tabs_container .nav-item');
```

Comment:
use targets

Reply:
We can't use targets because the ".nav-item" element is packaged inside the component: TabsContainerComponent

Reply:
ok just removed the id `#concept_tabs_container` and extract `.nav-item` as a stimulus value, with default equal to `.nav-item`

Reply:
We can't just adress .nav-item like this, this will target anything with the class .nav-item,
we added the id #concept_tabs_container explicitly for to target this, 
in case u want this code to be generalized to be used somewhere else I can pass that id as a value

Reply:
in this case `#concept_tabs_container` is equal to `this.element` the element on what the controller is connected.
So in this case you can replace 
```
 const tabsList = document.querySelectorAll('#concept_tabs_container .nav-item');
```
with
```
 const tabsList = this.element.querySelectorAll(".nav-item');
```